gulpfile.js

```diff
@@ -434,31 +468,31 @@ gulp.task('release', ['apps', 'clean-release'], function () {
         }
     });
 
-    var platforms = getPlatforms(true);
+    platforms = getPlatforms(true);
     console.log('Packing release.');
 
     if (platforms.indexOf('chromeos') !== -1) {
-        release_chromeos();
+        return release_chromeos();
```

Comment:
That's not right. If you do the return, only one platform can be generated each time. You must use the merge-stream again only for methods with gulp commands. In my PR you can see my code if you want.

Reply:
Damn, you are right, only one platform can be generated this way. I will sneak on your PR to see what you mean, just for lasiness ;-)